### ISSUE TITLE
Analysis Task: Non-Returning Functions - Discovered - java.lang.NullPointerException

### ISSUE THREAD

**Issue by BullyWiiPlaza** [NationalSecurityAgency/ghidra#1117]
**Describe the bug**
Loading a binary into `Ghidra` may yield the following exception:

```
Analysis Task: Non-Returning Functions - Discovered - 
java.lang.NullPointerException
	at ghidra.app.plugin.core.analysis.FindNoReturnFunctionsAnalyzer.checkNonReturningIndicators(FindNoReturnFunctionsAnalyzer.java:480)
	at ghidra.app.plugin.core.analysis.FindNoReturnFunctionsAnalyzer.detectNoReturn(FindNoReturnFunctionsAnalyzer.java:367)
	at ghidra.app.plugin.core.analysis.FindNoReturnFunctionsAnalyzer.added(FindNoReturnFunctionsAnalyzer.java:111)
	at ghidra.app.plugin.core.analysis.AnalysisScheduler.runAnalyzer(AnalysisScheduler.java:185)
	at ghidra.app.plugin.core.analysis.AnalysisTask.applyTo(AnalysisTask.java:39)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager$AnalysisTaskWrapper.run(AutoAnalysisManager.java:685)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:785)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:664)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:629)
	at ghidra.app.plugin.core.analysis.AnalysisBackgroundCommand.applyTo(AnalysisBackgroundCommand.java:62)
	at ghidra.framework.plugintool.mgr.BackgroundCommandTask.run(BackgroundCommandTask.java:101)
	at ghidra.framework.plugintool.mgr.ToolTaskManager.run(ToolTaskManager.java:315)
	at java.base/java.lang.Thread.run(Thread.java:830)

---------------------------------------------------
Build Date: 2019-Sep-23 1119 EDT
Ghidra Version: 9.1-BETA
Java Home: C:\Program Files\Java\jdk-13
JVM Version: Oracle Corporation 13-ea
OS: Windows 10 10.0 amd64
Workstation: 192.168.2.126
```

**To Reproduce**
1. Create a new Ghidra project
2. Import a file
3. Let Ghidra analyze the file
4. The exception about non-returning functions shows up in a dialog

**Expected behavior**
The exception does not occur.

**Environment (please complete the following information):**
 - OS: Windows 10 Pro 64-bit
 - Java Version: 13-ea
 - Ghidra Version: 9.1-DEV

**Comment by dev747368** [comment]
@BullyWiiPlaza - is this a locally compiled build, or the released 9.1 beta build?

**Comment by ryanmkurtz** [comment]
Fixed by 21b5bc3c33f0c2b1607db044b564f4c6864f8136
